plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/VarcharPredicatePushdownConfig.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.jdbc;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+
+public class VarcharPredicatePushdownConfig
+{
+    private boolean unsafePredicatePushdownEnabled;
+
+    public boolean isUnsafePredicatePushdownEnabled()
+    {
+        return unsafePredicatePushdownEnabled;
+    }
+
+    @Config("unsafe-varchar-predicate-pushdown.enabled")
+    @ConfigDescription("Enable potentially unsafe case insensitive predicate pushdown. Note that it may cause incorrect results.")
```

Comment:
```suggestion
    @ConfigDescription("Enable potentially unsafe predicate pushdown on char and varchar columns. Note that it may cause incorrect query results.")
```